cilium/cilium_helm_values.yaml

```diff
@@ -22,10 +22,12 @@ endpointRoutes:
 
 extraArgs:
   - --local-router-ipv4=169.254.23.0
+  - --local-router-ipv6=fe80::1
```

Comment:
I am just curious on where did you get this from?

Reply:
So for the local router ipv4 we use the link-local address which is that line above it. When I was talking with Will he mentioned that is something I would have to add and also needs to be added on the AKS side. 

I looked up what is used for an ipv6 link-local address and found here https://blogs.infoblox.com/ipv6-coe/fe80-1-is-a-perfectly-valid-ipv6-default-gateway-address/ that it needs to be an address with a header of fe80.